addons/paint-snap/lib/keyMirror.js

```diff
@@ -0,0 +1,3 @@
+const keyMirror = (obj) => Object.fromEntries(Object.entries(obj).map(([k]) => [k, k]));
```

Comment:
Don't think this is useful. I know this is a copy from vanilla, but don't need this ourselves.